src/components/EmailNotifications/EmailNotificationsForm.js

```diff
@@ -0,0 +1,302 @@
+import React, { useCallback, useState } from 'react'
+import styled from 'styled-components'
+import {
+  Button,
+  Checkbox,
+  Field,
+  GU,
+  IconCheck,
+  IconCross,
+  Link,
+  TextInput,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import LocalIdentityBadge from '../LocalIdentityBadge/LocalIdentityBadge'
+import { addressesEqual, transformAddresses } from '../../lib/web3-utils'
+import { validateEmail } from '../../utils/notifications-utils'
+
+import emailNotifcationIllustration from '../../../src/assets/emailNotifications.svg'
+
+function EmailNotificationsForm({
+  account,
```

Comment:
is it bad to pass a parameter instead of using a hook everywhere?

Reply:
Not necessarily, but if the parent doesn't need the prop directly (which I believe is the case here), then we should prefer using the hook.

Reply:
the parent needs it because we are doing the async calls from the parent now, so the parent needs to know the account

Reply:
I agree on calling `useWallet()` in components rather than passing it as a prop: I think we should consider it (and some others hooks holding state, like `useTheme()`) as being “part of the framework” in general, so that we don’t have to ask ourselves when to pass it vs. when to fetch it via the hook. Also it’s one less prop \o/